traffic_control/utils/send_traffic_sign_reals.py

```diff
@@ -67,18 +70,20 @@ def strip_or_none(prop):
 for feature in layer:
     print("Processing Feature ID: {0}".format(feature.GetFID()))
     feature_json = feature.ExportToJson(as_object=True)
-    geometry = feature_json["geometry"]
+    geometry = feature_json.get("geometry")
     point = ogr.Geometry(ogr.wkbPoint)
-    point.AddPoint_2D(geometry["coordinates"][0], geometry["coordinates"][1])
-    properties = feature_json["properties"]
+    point.AddPoint_2D(geometry.get("coordinates")[0], geometry.get("coordinates")[1])
+    properties = feature_json.get("properties")
     data = {
         "location": "SRID={0};{1}".format(SOURCE_SRID, point.ExportToWkt()),
-        "height": round(geometry["coordinates"][2], 6),
-        "legacy_code": strip_or_none(properties["type"]),
-        "txt": strip_or_none(properties["text"]),
-        "mount_type": strip_or_none(properties["mount_type"]),
-        "direction": strip_or_none(properties["direction"]),
-        "scanned_at": strip_or_none(properties["date"]),
+        "legacy_code": properties.get("type", None),
```

Comment:
Get will return None already as a default value, if key is not found. It's up to you if you want to change or not  :smile: https://docs.python.org/3/library/stdtypes.html#dict.get

Reply:
Original value was empty string as default value there, so easy to miss. Will definitely change that. Good catch!